src/styles/saas-list.less

```diff
@@ -48,12 +64,21 @@
       }
     }
     .card-footer {
-      display: table-cell;
```

Comment:
You could avoid the media settings here and just add 'hidden-xs" to the card footer html element.